### DISCUSSION TITLE
Question about the type of arguments like "vuContext", "events" "test" while using typescript

### DISCUSSION THREAD

**Discussion by ZhichengWangF1** [artilleryio/artillery#3225] [Artillery]
I am using Playwright with typescript. While using the functions as scenario of Artillery, I find that there are predefined arguments like "vuContext", "events" "test". However, using typescript, we should know the type of them to do the best pratice:
```
async function loginSearchAndLogout(page: Page, vuContext, events, test) {

}
```
For example, I know the type of "page" here is Page, then what about vuContext, events and test?

**Comment by hassy** [comment]
hi @ZhichengWangF1 👋 we don't provide typings for those arguments yet, but will look into it
